cmd/machine-api-operator/main.go

```diff
@@ -24,6 +27,8 @@ var (
 )
 
 func init() {
+	osconfigv1.AddToScheme(scheme.Scheme)
```

Comment:
this won't be picked here https://github.com/openshift/machine-api-operator/blob/master/pkg/operator/operator.go#L23
and https://github.com/openshift/machine-api-operator/blob/master/pkg/operator/operator.go#L85

Reply:
Danke